### ISSUE TITLE
How can I currently create a skeleton binding?

### ISSUE THREAD

**Issue by FischertBilligheim** [openhab/openhab2-addons#5549]
I tried to create a new binding for Openhab2 - but the existing documentation seems not be adapted to the current activities of the change to a new build-system.
So can anybody tell me how I  can create a skeleton of a binding - and how I can debug it in Eclipse?

That would help me really a lot. Thanks a lot in advance!

**Comment by davidgraeff** [comment]
The documentation is very clear on that actually: https://www.openhab.org/docs/developer/#setup-the-development-environment

At the moment you are on your own, the setup procedure is not yet finalized. It probably will be with OH 2.5m2. Stay tuned.

**Comment by FischertBilligheim** [comment]
Hello David,
the mentioned documentation does not work - even without working with  a Skeleton.

I'm using a windows10 machine, I did exactly what is described - I tried it with the existing serialbutton-binding - but I get 100 Errors in Eclipse:

![Openhab_Development](https://user-images.githubusercontent.com/20623089/57180577-4ee9d400-6e8a-11e9-91bc-211b0c98e835.PNG)

By the way: The description of installing the BndTools does not work - "all available sites" seems not be sufficiant. I installed the Tools directly from eclipse-marketplace.
